test/e2e/templates_test/templates_test.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2024 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build e2e || generic
```

Comment:
tests are showing under `atlas_generic_e2e` and it's variants, let's create one called `atlas_templates_e2e` in https://github.com/mongodb/mongodb-atlas-cli/blob/e1c9135ba67e664e05d372b7e02b09c4f93072ce/build/ci/evergreen.yml#L317 and add a tag `template`

Reply:
After an offline discussion with @fmenezes we decided to turn this e2e test into a tool and call the tool from a github action.